docs/src/token.mdx

```diff
@@ -80,46 +90,204 @@ Hardware Wallet URL (See [URL spec](https://docs.solana.com/wallet-guide/hardwar
 $ solana config set --keypair usb://ledger/
 ```
 
+  </TabItem>
+  <TabItem value="jsx" label="JS">
+
+#### Yarn
+```console
+yarn add @solana/spl-token
+```
+#### npm
+```console
+npm install @solana/spl-token
+```
+#### Browser bundle
+```html
+<script src="https://unpkg.com/@solana/spl-token@latest/lib/index.life.min.js"></script>
```

Comment:
`spl-token@latest/lib/index.life.min.js` -- will this link work? I don't know how unpkg works, does it automatically create this file?

Reply:
Oh, this no longer works. I'll update to remove it

Reply:
Removed in ed43e1d4010d8f77050477cdb6e5ed2093bc078a